src/data/terraform.json

```diff
@@ -104,25 +104,29 @@
 			"iconName": "docs",
 			"name": "CDKTF",
 			"path": "cdktf",
-			"productSlugForLoader": "terraform-cdk"
+			"productSlugForLoader": "terraform-cdk",
+			"hideSidebarTitle": true
```

Comment:
Adds newly introduced `hideSidebarTitle` prop for Terraform docs categories where it's needed.